doc/userguide/rules/intro.rst

```diff
@@ -87,11 +91,16 @@ you can pick from. These are:
 * http (either HTTP1 or HTTP2)
 * http1
 * http2
-* ftp
 * tls (this includes ssl)
```

Comment:
Should this list include all applayer protos that we support? If not, how is this defined? 
(asking cause I noticed `pgsql` isn't here. But not sure if it should. And if there are other missing)

Reply:
Hmm... Should this be looked into?